src/components/sbb-dialog/sbb-dialog.tsx

```diff
@@ -254,12 +256,21 @@ export class SbbDialog implements ComponentInterface {
 
   // Check if the pointerdown event target is triggered on the dialog.
   private _pointerDownListener = (event: PointerEvent): void => {
-    this._isPointerDownEventOnDialog = isEventOnElement(this._dialog, event);
+    this._isPointerDownEventOnDialog = event
+      .composedPath()
+      .filter((e) => e instanceof window.HTMLElement)
```

Comment:
Nitpick: You could use typescript assertion to not have to cast to HTMLElement in some
```suggestion
      .filter((e): e is HTMLElement => e instanceof window.HTMLElement)
```